src/cascadia/TerminalApp/CascadiaSettings.h

```diff
@@ -78,7 +80,7 @@ class TerminalApp::CascadiaSettings final
     static std::filesystem::path GetDefaultSettingsPath();
 
     const Profile* FindProfile(GUID profileGuid) const noexcept;
-    const ColorScheme* GetColorSchemeForProfile(const GUID profileGuid) const;
+    const winrt::TerminalApp::ColorScheme* GetColorSchemeForProfile(const GUID profileGuid) const;

```

Comment:
This should probably just be a `winrt::TerminalApp::ColorScheme` right? I'm pretty sure that pointers to the projected type are a no-no (@DHowett can correct me if I'm wrong)

Reply:
Absolutely a no-no